ReactWindows/ReactNative.Net46/Modules/Storage/AsyncStorageModule.cs

```diff
@@ -286,17 +283,19 @@ public void OnDestroy()
         private async Task<string> GetAsync(string key)
         {
             var storageFolder = await GetAsyncStorageFolder(false).ConfigureAwait(false);
-            if (storageFolder != null)
+            if (storageFolder == null)
+            {
+                return null;
+            }
+
+            var fileName = AsyncStorageHelpers.GetFileName(key);
+            if (storageFolder.CheckExistsAsync(fileName).Result == ExistenceCheckResult.FileExists)
```

Comment:
`.Result ` is used just to be consistent with similar calls in this code (like [here](https://github.com/ReactWindows/react-native-windows/blob/master/ReactWindows/ReactNative.Net46/Modules/Storage/AsyncStorageModule.cs#L353)). Would you like me to change it?